apparmor/apparmor.go

```diff
@@ -0,0 +1,110 @@
+package apparmor
+
+import (
+	"io"
+	"os"
+	"path"
+
+	"github.com/docker/libentitlement/templates"
+)
+
+var (
+	// profileDirectory is the file store for apparmor profiles and macros.
+	profileDirectory = "/etc/apparmor.d"
```

Comment:
is this the same for most platforms?  If we aren't sure, we should make this a `defaultProfileDirectory` and have a function that allows users to override this